action.yml

```diff
@@ -492,6 +492,26 @@ inputs:
     required: false
     default: "true"
 
+  rustfmt:
+    description: Enable or disable rustfmt
+    required: false
+    default: "false"
+  rustfmt_args:
+    description: Additional arguments to pass to the linter
+    required: false
+    default: " -- --color=never "
```

Comment:
```suggestion
    default: "-- --color=never"
```

Reply:
Fixed in https://github.com/wearerequired/lint-action/pull/585/commits/a71751672ccaa6093e1280f699e7d687109d1309